### DISCUSSION TITLE
Is it possible to change these names of the shutdown menu?

### DISCUSSION THREAD

**Discussion by willabytes** [Open-Shell/Open-Shell-Menu#1741] [Q&A] [answered]
Hi! When I compiled OpenShell from the source code I noticed that some of the menu names were "weird". For example, the shutdown menu. Instead of saying "Switch User", it says $Menu.SwitchUser. How can I change this? It also affected the settings names on the sidebar, but I managed to change them through the OpenShell settings menu. Help would be appreciated! :)

Image of the problem:
![bild](https://github.com/Open-Shell/Open-Shell-Menu/assets/29354273/6cdeb31f-a22e-4f93-ae41-bbe53a119ee3)

Best regards, W

**Comment by ge0rdi** [comment, marked as answer]
You need to copy `StartMenuL10N.ini` into the directory where your `StartMenuDLL.dll` is located.

**Reply by willabytes** [reply]
It worked! Thank you so much for your time. :)

**Reply by willabytes** [reply]
I really appreciate your work.
